scriptmodules/supplementary/emulationstation.sh

```diff
@@ -17,7 +17,8 @@ function depends_emulationstation() {
     getDepends \
         libboost-locale-dev libboost-system-dev libboost-filesystem-dev libboost-date-time-dev \
         libfreeimage-dev libfreetype6-dev libeigen3-dev libcurl4-openssl-dev \
-        libasound2-dev cmake libsdl2-dev libsm-dev
+        libasound2-dev cmake libsdl2-dev
```

Comment:
libsm-dev is required as a dependency as if it exists on the system cmake decides to link with it.